lib/stats-client.js

```diff
@@ -0,0 +1,35 @@
+var config = require('./config');
+var JSONAPIClient = require('json-api-client');
+var makeHTTPRequest = JSONAPIClient.makeHTTPRequest;
+
+statsClient = {
+  projectId: null,
+  workflowId: null,
+  
+  update: function (options) {
+    var keys = Object.keys(options);
+    for (var i in keys) {
+      this[keys[i]] = options[keys[i]];
+    }
+    return this
+  },
+
+  statCount: function (period, type) {
+    var query = this.workflowId ? "workflow_id=" + this.workflowId : "project_id=" + this.projectId;
```

Comment:
Why store workflow ID and project ID as properties on the client instead of passing them into the function? Makes it kind of unnecessarily stateful, right?